core/org.wso2.carbon.ui/src/main/java/org/wso2/carbon/ui/internal/CarbonUIServiceComponent.java

```diff
@@ -202,6 +178,32 @@ protected void activate(ComponentContext ctxt) {
         }
     }
 
+    private void printAdditionalContext(Context additionalContext, String adminConsoleURL, String webContextRoot) {
+        if ("/".equals(webContextRoot)) {
+            webContextRoot = "";
+        }
+        if (additionalContext != null && !"".equals(additionalContext.getContextName()) &&
```

Comment:
Shall we change this to !additionalContext.getContextName().isEmpty() ?

Reply:
That will require an additional null check. Since this is in startup process IMHO we can skip that bull check with this approach. WDYT?

Reply:
here, we have already done the null check, isn't it?

Reply:
Shouldn't we do additional null check for additionalContext.getContextName()?